src/kata1.fizzBuzz.js

```diff
@@ -1,5 +1,24 @@
 const fizzBuzz = (number) => {
+    const isDivisibleBy3 = number % 3 === 0;
+    const isDivisibleBy5 = number % 5 === 0;
+    const isDivisibleBy3And5 = number % 5 === 0  && number % 3 === 0;
 
-}
+ if(isDivisibleBy3And5)
+    {
+        return 'FizzBuzz'
+    }
+else if(isDivisibleBy3)
+    {
+        return 'Fizz'
+    }
+else if(isDivisibleBy5)
+    {
+        return 'Buzz'
+    }
+else
+    {
+        return number
+    }
```

Comment:
The approach here is correct, but you could probably make this more concise by using 3 if statements and then omitting the final else statement:

```js
const fizzBuzz = (number) => {
    const isDivisibleBy3 = number % 3 === 0;
    const isDivisibleBy5 = number % 5 === 0;
    const isDivisibleBy3And5 = number % 5 === 0 && number % 3 === 0;

    if(isDivisibleBy3And5) {
        return 'FizzBuzz';
    }
    if(isDivisibleBy3) {
        return 'Fizz';
    }
    if(isDivisibleBy5) {
        return 'Buzz';
    }

    return number;
}
```

Note also consistent use of semi colons and correct block syntax (opening curly brace should always trail).